example/ios/Podfile

```diff
@@ -49,6 +49,7 @@ target 'xmtpreactnativesdkexample' do
   flags = get_default_flags()
 
   pod 'secp256k1.swift', :modular_headers => true
+  pod 'RNCPushNotificationIOS', :path => '../node_modules/@react-native-community/push-notification-ios'
```

Comment:
this looks brittle - the pod doesn't get pulled in normally? i.e. via expo hooks checking npm packages

Reply:
It was something I pulled in from the docs here: https://reactnative.dev/docs/pushnotificationios.html, after removing it stuff still worked though so I don't think we need it. Removed in https://github.com/xmtp/xmtp-react-native/pull/31/commits/c97c0e93175ae126833d67cdb88eb467106a9900.

Reply:
Then actually removed in https://github.com/xmtp/xmtp-react-native/pull/31/commits/655762c2408345f07cc98364b21f4972fd49d309.